typescript/ecs/fargate-service-with-logging/index.ts

```diff
@@ -10,14 +10,12 @@ class WillkommenFargate extends cdk.Stack {
     const cluster = new ecs.Cluster(this, 'Ec2Cluster', { vpc });
 
     // create a task definition with CloudWatch Logs
-    const logging = new ecs.AwsLogDriver(this, "AppLogging", {
+    const logging = new ecs.AwsLogDriver({
       streamPrefix: "myapp",
     })
 
-    const taskDef = new ecs.FargateTaskDefinition(this, "MyTaskDefinition", {
-      memoryMiB: '512',
-      cpu: '256',
```

Comment:
this should be `cpu: 256`. Removing the quotes around 256 will fix the issue.

Reply:
@pkandasamy91 according to `FaragateTaskDefinition` the default `cpu` is 256 which is why I removed it.

Reply:
they are back